src/metatensor/models/experimental/soap_bpnn/train.py

```diff
@@ -116,6 +125,8 @@ def train(
     # Calculate and set the composition weights for all targets:
     logger.info("Calculating composition weights")
     for target_name in novel_capabilities.outputs.keys():
+        if "mts-models::aux::" in target_name:
+            continue
```

Comment:
Shouldn't this skip all outputs starting with `mts-models::`?

Reply:
Since `aux` is the name for non-trainable outputs, I think the `aux` ones should be skipped. In my mind, `mts-models::` would also be the name for arbitrary equivariant outputs (for which I have a rough design)